README.md

```diff
@@ -126,67 +121,6 @@ zebrad start
 See the [Installing Zebra](https://zebra.zfnd.org/user/install.html) and [Running Zebra](https://zebra.zfnd.org/user/run.html)
 sections in the book for more details.
 
-#### Optional Configs & Features
```

Comment:
Could we leave this section too, with just a link to the "install" page of the book?